### PR TITLE
#160451577 Article Report

### PR DESCRIPTION
**Pull request by obulaworld**
#### What does this PR do?
A user should be able to report an article

#### Description of Task to be completed?
- implement a functionality that allows a user to report an article
- Build out the report reducers to handle state update

#### How should this be manually tested?
- start the app
- select an article to view
- on the view article page, a report button is available
- click on the button, a modal will appear
- enter the reason for the report and click submit report

#### What are the relevant pivotal tracker stories?
[#160451577](https://www.pivotaltracker.com/story/show/160451577, "Users should be able to report an article")

#### Screenshots (if appropriate)

screen shot 2018-11-27 at 9 11 28 am

### CODE REVIEW

File: src/components/reusables/userDetail/_userDetail.scss [hunk 1]
```diff
@@ -21,3 +21,59 @@
...
+}
+
+.sidebar-close {
+  position: absolute;
+  top: 10rem;
+  right: 3rem;
+  font-size: 3.5rem;
+  color: #000;
+  cursor: pointer;
+}
```
Comment (mindsworth) [inline src/components/reusables/userDetail/_userDetail.scss]: Please add a new line here

**Review by mindsworth — APPROVED** [review]
Looking good to me...

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
#### Nitpick
Please implement feedback

File: src/components/reusables/userDetail/_userDetail.scss (hunk 1 above)
Comment (obulaworld) [reply src/components/reusables/userDetail/_userDetail.scss]: Implemented. Thanks

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
@obulaworld 
#### Critical 
Please rebase

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
@obulaworld 
Please rebase

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**